.github/workflows/cd.yml

```diff
@@ -236,10 +241,10 @@ jobs:
     steps:
       - name: Check environment
         run: |
-          if [ -n "${ENVIRONMENT}" ] && [ "${ENVIRONMENT}" != 'none' ] && [ "${ENVIRONMENT}" != 'dev' ] && [ "${BRANCH}" != 'main' ]; then
-            echo "Only 'dev' environment is allowed for non-main branches."
-            exit 1
-          fi
+          #if [ -n "${ENVIRONMENT}" ] && [ "${ENVIRONMENT}" != 'none' ] && [ "${ENVIRONMENT}" != 'dev' ] && [ "${BRANCH}" != 'main' ]; then
```

Comment:
remove commented code.

Reply:
Yes you are correct, I only added this for testing and forgot to remove it